dlt/common/schema/schema.py

```diff
@@ -392,7 +392,7 @@ def _coerce_non_null_value(self, table_columns: TTableSchemaColumns, table_name:
                 raise CannotCoerceColumnException(table_name, col_name, py_type, table_columns[col_name]["data_type"], v)
             # otherwise we must create variant extension to the table
             # pass final=True so no more auto-variants can be created recursively
-            # TODO: generate callback so DLT user can decide what to do
+            # TODO: generate callback sodltuser can decide what to do
```

Comment:
somehow surrounding spaces got eaten. and that happened in many places. I hope you can revert the commit and do that again?